modules/candidate_parameters/php/module.class.inc

```diff
@@ -33,4 +18,57 @@ class Module extends \Module
     {
         return "Candidate Parameters";
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @param string $type    The type of widgets to get.
+     * @param \User  $user    The user widgets are being retrieved for.
+     * @param array  $options A type dependent list of options to provide
+     *                        to the widget.
+     *
+     * @return \LORIS\GUI\Widget[]
+     */
+    public function getWidgets(string $type, \User $user, array $options) : array
+    {
+        switch($type) {
+        case 'candidate':
+            $factory = \NDB_Factory::singleton();
+            $baseurl = $factory->settings()->getBaseURL();
+
+            $candidate = $options['candidate'];
+            if ($candidate === null) {
+                return [];
+            }
+
+            // The candidate getConsents only returns the types of
+            // consents that the candidate has saved, we want to
+            // summarize all of the types configured in the database.
+            $consents = \Utility::getConsentList();
```

Comment:
```suggestion
    $consents = array_values(
        array_merge(
            \Utility:getConsentList(), 
            $candidate->getConsents()
        )
    );
```

Writing it this way makes it a bit more clear to me that  the $consents array is being sourced from multiple sources. Up to you though.

Reply:
I don't understand what you mean by multiple sources. The consent types are solely sourced from getConsentList and the data from the candidate.

Reply:
Yeah that's what I mean. Anyway it's mostly a cosmetic thing